automation/src/test/scala/org/broadinstitute/dsde/firecloud/api/Orchestration.scala

```diff
@@ -0,0 +1,185 @@
+package org.broadinstitute.dsde.firecloud.api
+
+import com.typesafe.scalalogging.LazyLogging
+import org.broadinstitute.dsde.firecloud.config.{AuthToken, Config}
+
+import scala.concurrent.duration._
+
+
+trait Orchestration extends FireCloudClient with LazyLogging {
+
+  def responseAsList(response: String): List[Map[String, Object]] = {
```

Comment:
I wonder if this type of thing could be baked into the `parseResponse` in the superclass.  So you could do something like
```
val response: List[String] = parseResponse[List[String]](getRequest(Config.FireCloud.orchApiUrl + "api/profile/billing"))
```